src/spikeinterface/sorters/launcher.py

```diff
@@ -26,14 +26,14 @@
     joblib=dict(n_jobs=-1, backend="loky"),
     processpoolexecutor=dict(max_workers=2, mp_context=None),
     dask=dict(client=None),
-    slurm=dict(tmp_script_folder=None, cpus_per_task=1, mem="1G"),
+    slurm={"tmp_script_folder": None, "sbatch_executable_path": "sbatch", "cpus-per-task": 1, "mem": "1G"},
```

Comment:
I would lean towards having something like 'sbatch_kwargs' as a sub-dict, with all kwargs that are passed directly to sbatch held there. The reason is that later `engine_kwargs` is iterated over an all parameters passed to `sbatch`. In future when editing people may miss this and not `pop` new arguments before `engine_kwargs` is iterated over, so accidentally pass unrelated arguments to `sbatch`. So could make sense to have `"sbatch_kwargs"` as a sub-dict to make the separation very clear then iterate only over `engine_kwargs["sbatch_kwargs"]`.

Reply:
This was what I had originally done, but then I thought it made the kwargs pretty cluttered and changed my mind. But I can see how that would help avoid mistakes in the future, I will change the code.